### PR TITLE
Add Cask for 'anonymous-vpn'

### PR DESCRIPTION
**Pull request by aren55555**
### Instructions

Delete the parts of this template that are not relevant (including these `Instructions`) and fill out the rest.  
If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.  
_Note:_ `{{cask_file}}` represents the cask file you’re submitting/editing.
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/anonymous-vpn.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'anonymous-vpn' do
```
Comment (vitorgalvao) [inline Casks/anonymous-vpn.rb]: `anonymousvpn` (no hyphen).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by aren55555** [comment]
@vitorgalvao so I know for the future should I be using the app name contained in the pkg `AnonymousVPN` or the sevice's name `Anonymous VPN` (w/ space, as used on their website & social media) to seed the token generation?

**Comment by aren55555** [comment]
@vitorgalvao should I also change the filename?

**Comment by vitorgalvao** [comment]
You should use the closest to the package itself, unless it’s a nonsensical name.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**